scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -177,7 +175,10 @@ sce_paths
 ```
 
 We can now read these files in and create a list of four SCE objects.
-Since `readr::read_rds()` can only operate on one input at a time (unlike `sqrt()` which can operate across a vector), we'll need to use `purrr::map()` to run it on all input file paths in one command.
+Since `readr::read_rds()` can only operate on one input at a time, we'll need to use `purrr::map()` to run it on all input file paths in one command.
+Although `sce_paths` is a vector (not a list), it will still work as input to `purrr:map()`.
+But, the output from this code will be a list, since that's what `purrr::map()` always returns.
+(We could convert `sce_paths` to a list first with `as.list(sce_paths)`, but for this example, the code will work just the same!)
```

Comment:
I don't think adding a suggestion of something not to do is particularly helpful here. I also don't like starting with a "but" if we don't really need to!
```suggestion
The output from this code will still be a list, since that's what `purrr::map()` always returns.
```